changelogs/current.yaml

```
@@ -156,6 +156,9 @@ bug_fixes:
 - area: redis
   change: |
     Fixed a bug where redis key with % in the key is failing with a validation error.
+- area: tracing
+  change: |
+    Fixed a bug in the Datadog tracer where Datadog's "operation name" would contain what should be in the "resource name."
```

```suggestion
    Fixed a bug in the Datadog tracer where Datadog's "operation name" field would contain what should be in the "resource name" field.
```

We have a format checker that requires these to end with a period and it apparently doesn't consider this to be correct. That's caused the CI failure.